components/ConversationContextMenu.tsx

```diff
@@ -0,0 +1,230 @@
+import TableView, { TableViewItemType } from "@components/TableView/TableView";
+import { ConversationReadOnly } from "@screens/ConversationReadOnly";
+import { backgroundColor } from "@styles/colors";
+import {
+  SIDE_MARGIN,
+  AUXILIARY_VIEW_MIN_HEIGHT,
+  HOLD_ITEM_TRANSFORM_DURATION,
+  contextMenuStyleGuide,
+  BACKDROP_DARK_BACKGROUND_COLOR,
+  BACKDROP_LIGHT_BACKGROUND_COLOR,
+  ITEM_WIDTH,
+} from "@utils/contextMenu/constants";
+import { BlurView } from "expo-blur";
+import React, { FC, memo, useCallback, useEffect } from "react";
+import {
+  Platform,
+  StyleSheet,
+  useColorScheme,
+  useWindowDimensions,
+  View,
+} from "react-native";
+import {
+  Gesture,
+  GestureDetector,
+  GestureHandlerRootView,
+} from "react-native-gesture-handler";
+import { Portal } from "react-native-paper";
+import Animated, {
+  runOnJS,
+  useAnimatedProps,
+  useAnimatedStyle,
+  useSharedValue,
+  withTiming,
+} from "react-native-reanimated";
+
+import { useIsSplitScreen } from "../screens/Navigation/navHelpers";
+
+const AnimatedBlurView =
+  Platform.OS === "ios"
+    ? Animated.createAnimatedComponent(BlurView)
+    : Animated.createAnimatedComponent(View);
```

Comment:
Should we make a BlurView component in `components` that we can reuse. So if later BlurView is fixed on Android we can just change it in one file

Reply:
Yes, good one, just pushed this change